torch/onnx/_constants.py

```diff
@@ -2,7 +2,7 @@
 
 ONNX_ARCHIVE_MODEL_PROTO_NAME = "__MODEL_PROTO"
 onnx_default_opset = 13
-onnx_main_opset = 16
```

Comment:
Do we make a note of newly updated main opsets anywhere in the documentation?

Reply:
Not that I know of. onnx_main_opset is an internal value that tracks the latest onnx opset version. Any suggestions?